symfony/src/Service/GoogleGroupsService.php

```diff
@@ -126,4 +126,18 @@ public function getRole(string $userEmail, string $groupEmail): ?string
             return null;
         }
     }
+
+    public function getMembers(string $groupEmail)
+    {
+        try {
+            $result = $this->callApi('members', 'listMembers', [$groupEmail]);
+            if ($result) {
+                return $result->getMembers();
+            } else {
+                return null;
+            }
+        } catch (GoogleException $e) {
+            return null;
+        }
```

Comment:
This try catch pattern is used in the getRole method above because we needed to handle the situation where the user is not in the group. In this case, we might need to think more about how we want to handle this if an error is returned. We might want to just not catch the exception at all, since anything other than a successful return of members is going to be an unexpected error.

At the very least, we should return an empty array instead of null if we want to silently fail.